decidim-admin/app/commands/decidim/admin/unhide_resource.rb

```diff
@@ -8,9 +8,10 @@ class UnhideResource < Decidim::Command
       #
       # reportable - A Decidim::Reportable
       # current_user - the user that performs the action
-      def initialize(reportable, current_user)
+      def initialize(reportable, current_user, with_admin_log: true)
```

Comment:
Let's update method definition ( the above comment)

Reply:
Done